_database-integrations/mysql/azure/microsoft-azure-mysql-v1.md

```diff
@@ -0,0 +1,173 @@
+---
+title: Microsoft Azure MySQL (v1)
+keywords: microsoft azure, azure, database integration, etl azure, azure etl
+permalink: /integrations/databases/microsoft-azure-mysql/v1
+summary: "Connect and replicate data from your Microsoft Azure SQL database using Stitch's Microsoft Azure MySQL integration."
+show-in-menus: true
+
+key: "microsoft-azure-mysql-integration"
+
+# -------------------------- #
+#     Integration Details    #
+# -------------------------- #
+
+name: "microsoft-azure-mysql"
+display_name: "Microsoft Azure MySQL"
+
+hosting-type: "microsoft-azure"
+
+this-version: "1"
+
+driver: |
+  [PyMySQL 0.9.3](https://pymysql.readthedocs.io/en/latest/){:target="new"}
+
+# -------------------------- #
+#       Stitch Details       #
+# -------------------------- #
+
+singer: true
+repo-url: https://github.com/singer-io/tap-mysql
+certified: true
+
+tap-name: "microsoft-azure-mysql"
+
+frequency: "30 minutes"
+tier: "Free"
+port: 3306
+db-type: "mysql"
+
+## Stitch features
+api-type: "platform.mysql"
+ssh: true
+ssl: true
+
+## General replication features
+
+anchor-scheduling: true
+cron-scheduling: true
+
+extraction-logs: true
+loading-reports: true
+
+table-selection: true
+column-selection: true
+table-level-reset: true
+
+## Replication methods
+
+define-replication-methods: true
+
+log-based-replication-minimum-version: "5.6.2"
+log-based-replication-master-instance: true
+log-based-replication-read-replica: true
+
+## Other Replication Methods
+
+key-based-incremental-replication: true
+full-table-replication: true
+
+view-replication: true
+
+
+# -------------------------- #
+#      Setup Requirements    #
+# -------------------------- #
+
+requirements-list:
+  - item: "**For Log-based replication:** Privileges in Azure that allow you to modify server parameters."
+  - item: |
+      **If using Log-based Incremental Replication**, you'll need the `ALTER DATABASE` privilege in {{ integration.display_name }}. This is required to complete the setup for Log-based Incremental Replication.
```

Comment:
I think this is leftover from the MSSQL doc you started with. It can come out entirely.